app/validators/positive_money_field_validator.rb

```diff
@@ -0,0 +1,15 @@
+class PositiveMoneyFieldValidator < ActiveModel::EachValidator
+  def validate_each(form, attribute, value)
+    if value.blank?
+      form.errors.add(attribute, form.error_msg_if_blank_or_zero)
+      return
+    end
+
+    unless value.to_s.match?(/\A-?\d+(?:\.\d+)?\z/)
```

Comment:
This is the numeric character check will match values such as "234", "1.2", "5.0" and "-30.45". Will not match values such as "1,000", "jdkfjsd", "1x", ".5" and "5."

Reply:
Yes `-30.45` should not be a valid input but we should show the other error message and not the "this is not a number" validation message, so it'll be caught but the other statement below.

I'll ask prod about these values "5. or .5 or 1,000"

Reply:
@arinchoi03 chatted with product and decided to allow these values as well, made the change here: https://github.com/codeforamerica/vita-min/pull/5947/commits/9b5062bc373b002b9ce174cc7f05e09852134d58 ; thanks for lifting that!